13/umbraco-commerce/.gitbook.yaml

```diff
@@ -12,3 +12,4 @@ getting-started/upgrading/version-specific-upgrades: upgrading/version-specific-
 how-to-guides/migrate-from-vendr-to-umbraco-commerce/migrate-custom-payment-providers: upgrading/migrate-from-vendr-to-umbraco-commerce/migrate-custom-payment-providers.md
 how-to-guides/migrate-from-vendr-to-umbraco-commerce/migrate-umbraco-commerce-checkout: upgrading/migrate-from-vendr-to-umbraco-commerce/migrate-umbraco-commerce-checkout.md
 how-to-guides/migrate-from-vendr-to-umbraco-commerce/README: upgrading/migrate-from-vendr-to-umbraco-commerce/README.md
+key-concepts/events: key-concepts/events/readme.md
```

Comment:
Oh! I actually thing that the readme file needs to be in all caps 🙈 
Remember something about this from the days with GitBook tests!

So, the file should be renamed to README.md 😅

Reply:
Got it.

Reply:
Yeah, so you also have to rename the file itself 😅

Reply:
I did it locally and somehow it dint show the changes in the commit. I did it via Github now, so the renamed file should be there 😳

Reply:
Ah, yes. For renaming files they need to be re-committed 😅

All looks good now though 👏🏼